bin/cassandra.in.sh

```diff
@@ -30,10 +30,10 @@ CLASSPATH="$CASSANDRA_CONF"
 # compiled classes. NOTE: This isn't needed by the startup script,
 # it's just used here in constructing the classpath.
 if [ -d $CASSANDRA_HOME/build ] ; then
-    #cassandra_bin="$CASSANDRA_HOME/build/classes/main"
-    cassandra_bin=`ls -1 $CASSANDRA_HOME/build/apache-cassandra*.jar`
+    #dse_db_bin="$CASSANDRA_HOME/build/classes/main"
+    dse_db_bin=`ls -1 $CASSANDRA_HOME/build/dse-db*.jar`
```

Comment:
we could fix this properly so that we use `find` and regular expression which would ensure we pick the right jar and not javadoc or sources

Reply:
That is true, but there may be some hidden assumptions. 
Eg. I can imagine upstream making a change that works with existing `ls` but not with more strict regexp

Reply:
agreed

Reply:
then I think we should fix this upstream